apps/namadillo/src/App/SignMessages/SignMessages.tsx

```diff
@@ -73,7 +72,7 @@ export const SignMessages = (): JSX.Element => {
                 value={data?.signature}
               />
             </Stack>
-            <ActionButton size="lg" borderRadius="sm" onClick={onCloseModal}>
+            <ActionButton borderRadius="sm" onClick={onCloseModal}>
```

Comment:
It's probably not worth fixing, but I am still getting the hairline at one particular zoom level on Chrome, both laptop screen and monitor. Not on Firefox though strangely.
![2024-07-15-134232_2880x2880_scrot](https://github.com/user-attachments/assets/79a09d6d-109d-41c5-90b9-d8399eff7bc5)

Reply:
I think this one might be really complex to fix :/ Maybe we should open an issue for that, but using so much time to fix this hairline that appears only when you zoom in the interface might be a bit too much

Reply:
Yeah, I think that is totally fine to leave :+1: